src/ontology/uberon.Makefile

```diff
@@ -734,7 +739,8 @@ $(TMPDIR)/GO.xrf_abbs: $(SRC)
 #  * species anatomy bridge axioms
 # This can be used to reveal both internal inconsistencies within uberon, and the improper linking of a species AO class to an uberon class with a taxon constraint
 $(TMPDIR)/uberon-edit-plus-tax-equivs.owl: $(OWLSRC) $(TMPDIR)/external-disjoints.owl $(TMPDIR)/bridges $(CATALOG_DYNAMIC)
-	$(OWLTOOLS_CAT_DYNAMIC) $< $(TMPDIR)/external-disjoints.owl `ls $(BRIDGEDIR)/uberon-bridge-to-*.owl | grep -v emap.owl` --merge-support-ontologies -o -f ofn $@
+	$(OWLTOOLS_CAT_DYNAMIC) $< $(TMPDIR)/external-disjoints.owl `ls $(BRIDGEDIR)/uberon-bridge-to-*.owl | grep -v emap.owl` --merge-support-ontologies -o -f ofn $@.tmp
+	$(ROBOT) expand -i $@.tmp -o $@
```

Comment:
This will need updating as this no longer applies cleanly now that the Makefile has been cleaned up. I can take care of that. The good news is that this step is now done entirely with ROBOT, so the whole dance of saving output OWLTools’s output in a temporary file that must then be processed by ROBOT will no longer be necessary.